core/services/gateway/connector/connector.go

```diff
@@ -71,24 +72,27 @@ func (c *gatewayConnector) Name() string { return c.lggr.Name() }
 
 type gatewayState struct {
 	conn     network.WSConnectionWrapper
-	config   ConnectorGatewayConfig
+	config   config.ConnectorGatewayConfig
 	url      *url.URL
 	wsClient network.WebSocketClient
 }
 
-func NewGatewayConnector(config *ConnectorConfig, signer Signer, handler GatewayConnectorHandler, clock clockwork.Clock, lggr logger.Logger) (GatewayConnector, error) {
-	if config == nil || signer == nil || handler == nil || clock == nil || lggr == nil {
+func NewGatewayConnector(configLocal config.GatewayConnectorConfig, signer Signer, handler GatewayConnectorHandler, clock clockwork.Clock, lggr logger.Logger) (GatewayConnector, error) {
```

Comment:
config *ConnectorConfig was a struct, now configLocal config.GatewayConnectorConfig is an interface